aptos-move/framework/move-stdlib/sources/option.move

```diff
@@ -145,9 +159,10 @@ module std::option {
     /// Aborts if `t` already holds a value
     public fun fill<Element>(t: &mut Option<Element>, e: Element) {
         let vec_ref = &mut t.vec;
-        if (vector::is_empty(vec_ref)) vector::push_back(vec_ref, e)
-        else abort EOPTION_IS_SET
+        if (vector::is_empty(vec_ref)) vector::push_back(vec_ref, e) else
```

Comment:
It's not entirely a bug, because this kind of statement doesn't have '{' and '}', so this situation does indeed occur. Ideally, the entire if-else statement can fit on the same line. However, for this specific case, we will add it to the test cases, and it is something that can be optimized.

Reply:
We fixed at https://github.com/movebit/movefmt/commit/d686c7270313e7ab0cba1f6e26b2445f28c28398